### ISSUE TITLE
[BUG] WCAG Navigation Freeze

### ISSUE THREAD

**Issue by jolevesq** [Canadian-Geospatial-Platform/geoview#1050]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When we tab to the map, the WCAG navigation window appears but once press enter on  Enable button, windows stays there for many seconds to a minutes before toggling the WCAG mode. This happen on footer demo page.

### Expected Behavior

Enable WCAG mode should be fast like on other pages

### Steps To Reproduce

1. Go to https://canadian-geospatial-platform.github.io/geoview/public/package-footer-panel.html
2. Tab to the map
3. Press Enter on Enable button
4. Wait for the WCAG mode to enable

### Anything else?

I doubt there is a correlation but the theme for this map is light and where I tested and it was fast was dark.

**Comment by jolevesq** [comment]
From latest React18 PR there is no freeze anymore
